client/daemon/storage/storage_manager.go

```diff
@@ -806,10 +807,20 @@ func (s *storageManager) TryGC() (bool, error) {
 		}
 		return true
 	})
-
 	quotaBytesExceed := totalNotMarkedSize - int64(s.storeOption.DiskGCThreshold)
 	quotaExceed := s.storeOption.DiskGCThreshold > 0 && quotaBytesExceed > 0
-	usageExceed, usageBytesExceed := s.diskUsageExceed()
+
+	metrics.DataUnReclaimedUsage.Set(float64(totalNotMarkedSize))
+	metrics.DataDiskGCThreshold.Set(float64(s.storeOption.DiskGCThreshold))
+	metrics.DataDiskGCThresholdPercent.Set(s.storeOption.DiskGCThresholdPercent)
+
```

Comment:
The `s.storeOption.DiskGCThresholdPercent` and `s.storeOption.DiskGCThreshold` is static, we should not update them every time.

Reply:
yes, this is used to record the threshold, so that we can display on the monitor, or there is better way to achieve that?